src/ray/raylet_client/BUILD.bazel

```diff
@@ -1,20 +1,55 @@
 load("//bazel:ray.bzl", "ray_cc_library")
 
-exports_files([
-    "raylet_client.h",
-])
+ray_cc_library(
```

Comment:
What’s the `visibility` on the new targets? De we need all the targets to be public?

Reply:
Most of them are intended to be public indeed, except for the "node_manager_client", which should be ":__subpackages__".